varcode/read_evidence/pileup_collection.py

```diff
@@ -585,3 +591,22 @@ def from_bam(pysam_samfile, loci):
             if close_on_completion:
                 pysam_samfile.close()
 
+def to_locus(variant_or_locus):
```

Comment:
I'm confused about why `variant` and `locus` are combined here?

(In general, I feel like `x_or_y` isn't good practice, but I don't yet understand why the code wants this in the first place.)